code/obj/decorations.dm

```diff
@@ -263,15 +277,47 @@
 		src.take_damage(W.force)
 		user.visible_message("<span class='alert'><b>[user] hacks at [src] with [W]!</b></span>")
 
+	proc/graze(mob/user)
+		src.bites -= 1
+		var/desired_mask = (src.bites / initial(src.bites)) * 5
+		desired_mask = round(desired_mask)
+		desired_mask = max(1,desired_mask)
+		desired_mask = min(desired_mask, 5)
+
+		if (desired_mask != current_mask)
+			current_mask = desired_mask
+			src.add_filter("bite", 0, alpha_mask_filter(icon=icon('icons/obj/foodNdrink/food.dmi', "eating[desired_mask]")))
+
+		eat_twitch(user)
+		playsound(user, "sound/items/eatfood.ogg", rand(10,50), 1)
+
+		if (is_plastic)
+			user.setStatus("weakened", 3 SECONDS)
+			user.visible_message("<span class='notice'>[user] takes a bite out of [src] and chokes on the plastic leaves.</span>", "<span class='alert'>You munch on some of [src]'s leaves, but realise too late it's made of plastic. You start choking!</span>")
+			user.take_oxygen_deprivation(20)
+			user.losebreath+=2
+		else
+			user.changeStatus("food_hp_up", 20 SECONDS)
+			user.visible_message("<span class='notice'>[user] takes a bite out of [src].</span>", "<span class='notice'>You munch on some of [src]'s leaves, like any normal human would.</span>")
+			var/mob/living/carbon/human/H = user
```

Comment:
why is it a human

Reply:
I'm checking for "sims" on the next line, which can only be found on humans. Is there another way i can go about this?

Reply:
How do you know it's a human you didn't check

Reply:
moreso the argument of the proc doesn't specify that it only takes humans

Reply:
Alright yeah that makes more sense

Reply:
unnecessary now